src/main/java/com/wishlist/exceptions/GlobalExceptionHandler.java

```diff
@@ -0,0 +1,17 @@
+package com.wishlist.exceptions;
+
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.ControllerAdvice;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+
+@ControllerAdvice
+@Slf4j
+public class GlobalExceptionHandler {
```

Comment:
It is a good idea to handle all business exceptions in one place.

Reply:
Or at least in few places, but one exception should always be handled in the same way.